specification.md

```diff
@@ -194,6 +208,32 @@ Represents person registration data.
 }
 ```
 
+### RegistrationInfo
+
+Represents information related to when and how to register for the competition.
+
+| Attribute | Type | Description |
+| --- | --- | --- |
+| `openTime` | [`DateTime`](#datetime) | The point in time when online registration opens. |
+| `closeTime` | [`DateTime`](#datetime) | The point in time when online registration closes. |
+| `baseEntryFee` | `Integer` | The competition's base fee for online registration. |
+| `currencyCode` | [`CurrencyCode`](#currencycode) | The currency of the `baseEntryFee`. |
+| `competitorLimit` | `Integer` | The maximum number of competitors who can register for this competition. |
+| `walkInsPermitted` | `Boolean` | Whether non-registered competitors are permitted to sign up at the competition. +
+
+#### Example
+
+```json
+{
+  "openTime": "2023-08-29T05:00:00Z",
+  "closeTime": "2023-11-18T05:00:00Z",
+  "baseEntryFee": 20,
+  "currencyCode": "USD",
+  "competitorLimit": 100,
+  "walkInsPermitted": false,
```

Comment:
thoughts on just continuing with maintaining "on the spot" language with something like `onTheSpotRegistrationAccepted`?

Reply:
Good point, I forgot we had a term for this already. Fixed.